locksmith/src/fulfillment/dispatcher.ts

```diff
@@ -14,7 +18,7 @@ interface KeyToGrant {
 export default class Dispatcher {
   async getPurchaser(network: number) {
     const provider = new ethers.providers.JsonRpcProvider(
-      networks[network].publicProvider
+      networks[network].provider
```

Comment:
So, when sending transactions, it's better to use the `publicProvider` recommended by the networks themselves.
In the past we have seen issues where some of our transactions were queued because they were sent from Alchemy.

Reply:
oh, ok. I thought public ones get rate limited or have lower throughout so I changed to using paid ones. Will revert.

Reply:
yeah when "reading" we want to use ours when when sending tx we want to use the public ones.